silkworm/silkrpc/commands/ots_api.cpp

```diff
@@ -966,7 +966,7 @@ ChunkProvider::ChunkProvider(silkworm::rpc::ethdb::Cursor* cursor, evmc::address
     first_seek_key_value_ = first_seek_key_value;
 }
 
-Task<ChunkLocatorResponse> ChunkLocator::get(uint64_t min_block) {
+Task<ChunkLocatorResponse> ChunkLocator::get(BlockNum min_block) {
```

Comment:
The same change should be applied also in `ots_api.hpp`

Reply:
ok